### PR TITLE
docs(windows): update for windows-py3 [signed-commit]

### PR DESCRIPTION
**Pull request by noelmcloughlin**
### What does this PR do?

Update README giving ONE example of WINDOWS bootstrapping
Default to python3 instead of python2

Future PRs can further elaborate the README, but this PR is minimal update for Windows

### What issues does this PR fix or reference?
n/a

### Previous Behavior

The README gives little clue how to proceed on WINDOWS (no examples)

### New Behavior

An example using curl / powershell is added to README

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -227,6 +227,20 @@ Installing the latest master branch of Salt:
...
 
+Install on Windows
+~~~~~~~~~~~~~~~~~~
+
+Using ``curl.exe`` to install latest development version from GitHub:
+
+.. code:: console
+
+  curl.exe -o bootstrap-salt.ps1 -L 
+  https://raw.githubusercontent.com/saltstack/salt-bootstrap/develop/bootstrap-salt.ps1
```
Comment (bryceml) [inline README.rst]: https://winbootstrap.saltstack.com is what should be used.

**Comment by bryceml** [comment]
what is all the other stuff? why is the file named bob that you added?

File: README.rst (hunk 1 above)
Comment (twangboy) [reply README.rst]: And there is no `curl` on Windows by default. You should probably give some powershell code here:
```
Invoke-WebRequest `
    -Uri https://raw.githubusercontent.com/saltstack/salt-bootstrap/develop/bootstrap-salt.ps1 `
    -OutFile C:\Temp\bootstrap-salt.ps1
```

**Review by twangboy — CHANGES_REQUESTED** [review]
Whey are we including Chocolatey scripts here?

File: README.rst (hunk 1 above)
Comment (noelmcloughlin) [reply README.rst]: Thanks @bryceml   that is better.

Comment (noelmcloughlin) [reply README.rst]: Good point @twangboy .. I was using cygwin which includes curl.  
I added a second example. comments welcome as I'm not familiar with PS.

**Comment by noelmcloughlin** [comment]
> what is all the other stuff? why is the file named bob that you added?

Sorry - I got rid of that.

**Comment by noelmcloughlin** [comment]
> Whey are we including Chocolatey scripts here?

I removed the other two files.

File: README.rst [hunk 2]
```diff
@@ -227,6 +227,31 @@ Installing the latest master branch of Salt:
...
+
+  Invoke-WebRequest `
+    -Uri https://winbootstrap.saltstack.com
+    -OutFile C:\Temp\bootstrap-salt.ps1
+  Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser
+  C:\Temp\bootstrap-salt.ps1
+  Set-ExecutionPolicy -ExecutionPolicy Undefined -Scope CurrentUser
+
+
+Using ``cygwin`` to install latest development version from GitHub:
```
Comment (bryceml) [inline README.rst]: to install the development version, it's https://winbootstrap.saltstack.com/develop  without the /develop you get the stable version.

also, it's not "from Github" if it's not a github url is it?

Comment (bryceml) [reply README.rst]: I would recommend using the stable version in most circumstances.

File: bootstrap-salt.ps1 [hunk 3]
```diff
@@ -85,7 +85,7 @@ Param(
     [Parameter(Mandatory=$false,ValueFromPipeline=$true)]
     # Doesn't support Python versions prior to "2017.7.0"
     [ValidateSet("2","3")]
-    [string]$pythonVersion = "2",
+    [string]$pythonVersion = "3",
```
Comment (bryceml) [inline bootstrap-salt.ps1:88]: @twangboy do we need a less interruptive way of changing this? or is this fine?

File: README.rst [hunk 4]
```diff
@@ -227,6 +227,31 @@ Installing the latest master branch of Salt:
   curl -L https://bootstrap.saltstack.com | sudo sh -s -- git master
 
 
+Install on Windows
+~~~~~~~~~~~~~~~~~~
+
+Using ``PowerShell`` to install latest development version from GitHub:
```
Comment (bryceml) [inline README.rst]: same as line 245.

File: README.rst (hunk 2 above)
Comment (noelmcloughlin) [reply README.rst]: Thanks, I have updated commit  for these comments.

File: README.rst [hunk 5]
```diff
@@ -227,6 +227,31 @@ Installing the latest master branch of Salt:
...
 
+Install on Windows
+~~~~~~~~~~~~~~~~~~
+
+Using ``PowerShell`` to install latest stable version:
+
+.. code:: console
+
+  Invoke-WebRequest `
+    -Uri https://winbootstrap.saltstack.com
```
Comment (bryceml) [inline README.rst]: looks like a missing ` on the end of this line.

Comment (noelmcloughlin) [reply README.rst]: Thanks I checked Invoke-WebRequest syntax and fixed that line.

File: bootstrap-salt.ps1 (hunk 3 above)
Comment (twangboy) [reply bootstrap-salt.ps1:88]: That's fine. It's just changing the default.

**Review by bryceml — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**